views/js/runner/plugins/controls/progressbar/progressbar.js

```diff
@@ -162,8 +170,10 @@ define([
             var progressIndicator = config.type || 'percentage';
             var progressScope = config.scope || 'test';
 
+            progressShowTotal = !!config.showTotal;
```

Comment:
As the config becomes more complex, you should probably transform the two last parameters of the `progressUpdater()` function into a config set:
```javascript
...
function progressUpdater(testContext, testMap, progressConfig) {
    ...
}
...
var progressConfig = {
    indicator: config.type || 'percentage',
    scope: config.scope || 'test',
    showTotal: !!config.showTotal
};
...
var progressData = progressUpdater(testRunner.getTestContext(), testRunner.getTestMap(), progressConfig);
...
```